run.py

```diff
@@ -122,10 +127,11 @@ def model_state(country, state, interventions=None):
     },
 ]
 
-Dataset = CDSDataset()
+Dataset = _Dataset()
```

Comment:
Can we try to reserve capitalized variables for ClassNames and change this variable from `Dataset` -> `dataset`

I realize this was there before, but helps make it clear when you're reading code whether or not you're accessing a static/classmethod `Dataset.get_something` vs an instance method `dataset.get_something`.

Reply:
I saw this. I was trying not to fix too many things I wasn't touching :) I can refactor this.

Reply:
I dropped these changes from this pr.